doc/ref_cert/RC1/chapters/chapter02.md

```diff
@@ -96,16 +95,14 @@ The Infrastructure Profile Catalog contains the following attributes:
 * Profiles are offered to VNFs as an instance types with predefined compute flavors.
   * A particular set of options is an instance type
   * Compute flavors: .tiny, .small etc as defined in [RM §4.2.1.1](../../../ref_model/chapters/chapter04.md#4211-predefined-compute-flavours)
-* NFVI performance profiles, for which NFVI validations will support and be verified against, are defined as basic and network intensive. <!-- , and compute intensive. --> Details for each of these profiles can be found in [RM §2.3](../../../ref_model/chapters/chapter02.md#2.3).
+* NFVI performance profiles, for which NFVI validations will support and be verified against, are defined as basic and network intensive. Details for each of these profiles can be found in [RM §2.3](../../../ref_model/chapters/chapter02.md#2.3).
 <p align="center"><img src="../figures/RC_NFVI_Profiles.png" alt="NFVI Profiles" title="NFVI Profiles" width="100%"/></p>
 <p align="center"><b>Figure 2-2:</b> NFVI Profiles</p>
-<!---Targeted VNF Classes/Families for baseline measurements are described in chapter XXXX.--->
 
 <a name="2.4"></a>
 ## 2.4 Profiles Reference
 Different vendors have different types of VNFs to serve different use-cases. A VNF like Broadband Network Gateway (BNG) would require high networking throughout whereas a VNF like Mobility Management Entity (MME) would require high computing performance. As such, BNG would require high KPI values for network throughput and MME would require high CPU performance KPIs like Index Score, Instructions Per Second (IPS) etc. The target NFVI to cater these needs would have different characteristics. Depending on VNF's requirements, the NFVI can be categorized into below profiles:
 * Basic (B) profile for standard computing and
-<!-- * Compute intensive (C) profile where predictable computing performance is expected and -->
 * Network intensive (N) profile offering predictable computing performance along with low latency and high networking throughout
```

Comment:
high networking throughout? Or should it be high networking throughput?
```suggestion
* Network intensive (N) profile offering predictable computing performance along with low latency and high networking throughput
```

Reply:
ok let's merge your proposal in a second change. It takes too much time for a nit.